src/components/Interpretations/common/Message/MessageIconButton.js

```diff
@@ -51,6 +51,20 @@ const MessageIconButton = ({
                         color: ${colors.teal600};
                         font-weight: 500;
                     }
+                    .button:hover {
```

Comment:
Wondering if the selected button should have a stronger selected color (teal) rather than grey?

Reply:
Changed the hover color to a shade of teal